addons/web/static/src/views/kanban/kanban_column_quick_create.xml

```diff
@@ -15,9 +15,10 @@
                             class="form-control o_input bg-transparent fs-4"
                             t-attf-placeholder="{{ relatedFieldName }}..."
                             t-ref="autofocus"
-                            t-model.trim="state.columnTitle"
+                            t-model="state.columnTitle"
                             t-on-focus="() => state.hasInputFocused = true"
                             t-on-blur="() => state.hasInputFocused = false"
+                            t-on-keydown="(ev) => ev.key === 'Enter' and this.validate()"
```

Comment:
suggestion: t-on-keydown="onInputKeydown" and implement the logic in JS, not in a single line in the template. That would allow you to read the content of the input as well, trim it, and give it to validate